src/azure-cli-core/azure/cli/core/commands/_introspection.py

```diff
@@ -71,6 +71,9 @@ def extract_args_from_signature(operation, expose_raw_as_no_wait=False):
     excluded_params = list(EXCLUDED_PARAMS)
     if not expose_raw_as_no_wait:
         excluded_params.append('raw')
+    elif not 'raw' in args:
+        raise ValueError("There is no 'raw' argument exposed for no-wait")
```

Comment:
This would confuse me if I were not familiar with the implementation of no-wait. I would suggest something like "Unable to enable no-wait option. Operation '{}' does not have a 'raw' parameter.".format(operation.name)

In other words, make it clear that the method must have the 'raw' capability in order to expose no-wait through this argument.